app/services/folio_client.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'http'
+
+class FolioClient
+  DEFAULT_HEADERS = {
+    accept: 'application/json, text/plain',
```

Comment:
If we're going to parse JSON from the response, then `text/plain` isn't really acceptable to us.